service/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package service exposes core UI and API endpoints.
```

Comment:
Can we say it's the legacy API?

Is it worth mentioning that most API endpoints live in `rest/route` nowadays?

Reply:
I don't think that's true. Many of the API endpoints are still core parts of Evergreen, used by the agent, for example.